src/ethereum_test_fixtures/blockchain.py

```diff
@@ -32,7 +40,16 @@
     ZeroPaddedHexNumber,
 )
 from ethereum_test_exceptions import EngineAPIError, ExceptionInstanceOrList
-from ethereum_test_forks import Fork, Paris
+from ethereum_test_forks import (
+    Cancun,
+    Constantinople,
+    Fork,
+    Frontier,
+    Homestead,
+    Istanbul,
+    Paris,
+    Prague,
+)
```

Comment:
We could use `get_forks` to make the fork resolution by name more dynamic.
```suggestion
from ethereum_test_forks import Fork, Paris, get_forks
```

And then simply define something like:
```python
all_forks_by_name = {fork.name(): fork for fork in get_forks()}
```